jobs/config.json

```diff
@@ -4651,6 +4651,28 @@
       "sig-network"
     ]
   },
+  "ci-kubernetes-e2e-gci-gce-ipvs": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gci-gce-ipvs.env",
+      "--extract=ci/latest",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=ubuntu",
+      "--gcp-project=k8s-jenkins-gce-gci-ipvs",
```

Comment:
I don't think we need a specific project for this. You can probably remove this.

@krzyzacy Thoughts?

Reply:
use the ingress pool :-)

Reply:
This isn't an ingress test :) 

We don't have a dedicated project for ipvs anyway

Reply:
yeah, then just drop the flag if the test can work with default quota :-)

Reply:
done